packages/web/src/components/collection/desktop/CollectionHeader.tsx

```diff
@@ -243,7 +243,17 @@ export const CollectionHeader = (props: CollectionHeaderProps) => {
             {isLoading ? (
               <Skeleton height='24px' width='150px' />
             ) : userId !== null ? (
-              <Text variant='title' strength='weak' tag='h2' textAlign='left'>
+              <Text
```

Comment:
wrapping in flex no good?

Reply:
I wanted to keep the Text props/styles that were being passed down